journal/src/main/scala/com/evolutiongaming/kafka/journal/ConsumeActionRecords.scala

```diff
@@ -18,14 +22,18 @@ trait ConsumeActionRecords[F[_]] {
 
 object ConsumeActionRecords {
 
-  def apply[F[_] : BracketThrowable](
-    consumer: Resource[F, Journals.Consumer[F]],
-    log: Log[F])(implicit
-    consRecordToActionRecord: ConsRecordToActionRecord[F]
+  def apply[F[_] : BracketThrowable: Clock](
+    consumerPool: ResourcePool[F, Journals.Consumer[F]],
+    log: Log[F],
+    poolMetrics: ConsumerPoolMetrics[F],
+  )(implicit
+    consRecordToActionRecord: ConsRecordToActionRecord[F],
+    genConcurrent: GenConcurrent[F, Throwable]
```

Comment:
I wonder why such parameter is required? Why not just `Concurrent[F]` instead of `GenConcurrent[F, Throwable]`?

Also, why named parameter is required, would not it make sense to just replace `F[_] : BracketThrowable` with `F[_]: Concurrent`? :thinking:

Reply:
I see you fixed this in `ConsumeActionRecords.of`. May be the same fix could also be done in `ResourcePool.fixedSize`?